repo/content/info.go

```diff
@@ -6,6 +6,18 @@ import (
 	"github.com/kopia/kopia/repo/blob"
 )
 
+// operation indicates the life-cycle state of a content
+type operation uint8
+
+// Content write operations corresponding to create, delete, undelete
+const (
+	// creating is the 0 value for this enum
+	// nolint:deadcode,varcheck
+	creating operation = iota
```

Comment:
the reason for the constant being called `creating` instead of `create` (operation) is that `delete` is a built-in.